packages/react-instantsearch-hooks-dom/src/index.ts

```diff
@@ -1,2 +1,3 @@
 export * from 'react-instantsearch-hooks';
 export * from './widgets/Hits';
+export * from './widgets';
```

Comment:
maybe add Hits to the barrel in this PR too then

Reply:
I wasn't sure you were going with it, but fine by me.

Reply:
Addressed in 45324117.